noetic/distribution.yaml

```
@@ -1017,6 +1017,21 @@ repositories:
       url: https://github.com/ros/message_runtime.git
       version: kinetic-devel
     status: maintained
+  mrpt2:
+    doc:
+      type: git
+      url: https://github.com/MRPT/mrpt.git
```

```suggestion
      url: https://github.com/mrpt/mrpt.git
```

Just for consistency with the source information.